carapace-server/src/main/java/org/carapaceproxy/configstore/HerdDBConfigurationStore.java

```diff
@@ -85,12 +85,12 @@
     private static final String DIGITAL_CERTIFICATES_TABLE_NAME = "digital_certificates";
     private static final String CREATE_DIGITAL_CERTIFICATES_TABLE = "CREATE TABLE " + DIGITAL_CERTIFICATES_TABLE_NAME
             + "(domain string primary key, privateKey string, chain string, "
-            + "state string, pendingOrder string, pendingChallenge string, available tinyint)";
+            + "state string, pendingOrder string, pendingChallenge string)";
```

Comment:
You have deleted a field from the table. For those who already have that table full, it is necessary to make that field nullable first. Otherwise the insertion fails

Reply:
@pv3nturi I just did a manual check  and by default the column are created nullable. 
```
INSERT INTO  digital_certificates (domain, privateKey, chain, state, pendingOrder, pendingChallenge) values ('example.com', 'test', 'test', 'test', 'test', 'test')
```

```
| domain   | privatekey | chain | state | pendingorder | pendingchallenge | available |
| ------   | ---------- | ----- | ----- | ------------ | ---------------- | --------- |
| example.com | test| test| test| test| test| <NULL>    |
```